### PR TITLE
Commerce 5483 dxp 1of5

### PR DESCRIPTION
**Pull request by garofalog**

### CODE REVIEW

File: modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/test/dev/public/js/static-env-utils.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+		window.addEventListener(name, fn);
+	},
+	staticEnvTestUtils: {
+		print: (message, type) => ({message, type}),
+	},
+};
+
+window.defaultFetch = fetch;
+
+// window.fetch = (resource, {headers, ...init}) => {
```
Comment (andrea-ale-sbarra) [inline modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/test/dev/public/js/static-env-utils.js:62]: remove comments

File: modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/webpack.config.dev.js [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const path = require('path');
+const webpack = require('webpack');
+
+// const components = require('./test/dev/components/index');
```
Comment (andrea-ale-sbarra) [inline modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/webpack.config.dev.js:19]: remove comments

File: modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/webpack.config.dev.js [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+// const { defineServerResponses } = require('./test/dev/fakeServerUtilities');
+
+const outputPath = path.resolve(__dirname, './dev/public');
+
+// function getComponentPath(entry) {
```
Comment (andrea-ale-sbarra) [inline modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/webpack.config.dev.js:24]: remove comments

File: modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/webpack.config.dev.js [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+// 	return path.join(__dirname, 'test', 'dev', 'components', entry);
+// }
+
+// eslint-disable-next-line no-undef
```
Comment (andrea-ale-sbarra) [inline modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/webpack.config.dev.js:28]: remove comments

File: modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/webpack.config.dev.js [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+module.exports = {
+	devServer: {
+
+		// before(app) {
```
Comment (andrea-ale-sbarra) [inline modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/webpack.config.dev.js:32]: remove comments

**Review by andrea-ale-sbarra — CHANGES_REQUESTED** [review]

**Comment by garofalog** [comment]
ci:test:sf

**Comment by garofalog** [comment]
ci:test:sf

**Comment by garofalog** [comment]
ci:test:sf

**Comment by garofalog** [comment]
ci:test:sf

**Comment by garofalog** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by garofalog** [comment]
ci:test:sf

**Comment by garofalog** [comment]
ci:test:relevant

**Comment by garofalog** [comment]
ci:test:relevant

File: modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/bnd.bnd [hunk 6]
```diff
@@ -0,0 +1,6 @@
+Bundle-Name: Liferay Commerce Shop By Diagram Web
+Bundle-SymbolicName: com.liferay.commerce.shop.by.diagram.web
+Bundle-Version: 5.0.2
```
Comment (Commando18) [inline modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/bnd.bnd:3]: 1.0.0

File: modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/build.grandle [hunk 7]
```diff
@@ -0,0 +1,11 @@
+dependencies {
+	compileOnly group: "com.liferay.portal", name: "com.liferay.portal.kernel", version: "default"
+	compileOnly group: "com.liferay.portal", name: "com.liferay.util.taglib", version: "default"
+	compileOnly group: "javax.portlet", name: "portlet-api", version: "3.0.1"
+	compileOnly group: "javax.servlet.jsp", name: "javax.servlet.jsp-api", version: "2.3.1"
+	compileOnly group: "org.osgi", name: "org.osgi.service.component.annotations", version: "1.3.0"
+	compileOnly group: "org.osgi", name: "org.osgi.service.jaxrs", version: "1.0.0"
+	compileOnly group: "org.osgi", name: "osgi.core", version: "6.0.0"
+	compileOnly project(":apps:frontend-taglib:frontend-taglib")
+	compileOnly project(":apps:frontend-taglib:frontend-taglib-react")
```
Comment (Commando18) [inline modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/build.grandle:10]: Delete those dependencies, we don't need it for the moment

File: modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/style/icons.svg [hunk 8]
```diff
@@ -0,0 +1,1914 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Commando18) [inline modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/style/icons.svg:1]: Do we really need this file?

File: modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/build.grandle (hunk 7 above)
Comment (Commando18) [reply modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/build.grandle]: Rename file to build.gradle

**Comment by Commando18** [comment]
@garofalog please squash commits

File: modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/style/icons.svg (hunk 8 above)
Comment (garofalog) [reply modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/style/icons.svg:1]: yep

**Comment by garofalog** [comment]
ci:test:relevant

**Comment by garofalog** [comment]
ci:test:sf

**Comment by garofalog** [comment]
ci:test:sf

**Pull request closed without merging**